library/std/src/panicking.rs

```diff
@@ -296,10 +296,17 @@ fn default_hook(info: &PanicInfo<'_>) {
             }
             Some(BacktraceStyle::Off) => {
                 if FIRST_PANIC.swap(false, Ordering::SeqCst) {
-                    let _ = writeln!(
-                        err,
-                        "note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace"
-                    );
+                    let _ = if let Ok(_) = crate::env::var("PSModulePath") {
+                        writeln!(
+                            err,
+                            "note: run with `$env:RUST_BACKTRACE=1` environment variable to display a backtrace."
+                        )
+                    } else {
+                        writeln!(
+                            err,
+                            "note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace"
```

Comment:
`"note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace."`

Reply:
oof, add a period the end.

Reply:
https://github.com/rust-lang/rust/blob/master/library/std/src/panicking.rs#L301

The original msg doesn't seem to include a period at the end, so I think I should *remove* them instead :)

Reply:
Hm, that's odd, the RUST_BACKTRACE=full msg does include a period.

Meh doesn't really matter I'll add it.